qa/tasks/deepsea.py

```diff
@@ -861,6 +876,59 @@ def teardown(self):
         pass
 
 
+class InstallMigrationRPM(DeepSea):
+
+    err_prefix = "(install_migration_rpm subtask) "
+
+    # rpm_location = "http://download.suse.de/ibs/Devel:/Storage:/5.0:/Testing/images/x86_64/"
+    rpm_location = ("http://51.68.80.25/artifacts/ci/"
+                    "smithfarm:Devel:Storage:5.0:Testing-images/x86_64/")
```

Comment:
@tserong @gekios Note how we are hard-coding the location of the Migration RPM, and it's a Provo (ECP) mirror of Devel:Storage:5.0:Testing - http://51.68.80.25/artifacts/ci/smithfarm:Devel:Storage:5.0:Testing-images/x86_64/ - which contains the version `SLES15-Migration-1.15.1-8.x86_64.rpm`

Whereas the *actual* Devel:Storage:5.0:Testing repo - http://download.suse.de/ibs/Devel:/Storage:/5.0:/Testing/images/x86_64/ - contains a newer version `SLES15-Migration-1.15.2.ses.0-15.x86_64.rpm`

But, in any case, we will need to modify the code in the `InstallMigrationRPM` class once the migration RPM is available via the update channel. Any idea when that will be?

Reply:
> In any case, we will need to modify the code in the `InstallMigrationRPM` class once the migration RPM is available via the update channel. Any idea when that will be?

Not sure yet. MRs were submitted last week (https://build.suse.de/request/show/201042, https://build.suse.de/request/show/201043) and reviewed OK but not yet accepted by the maintenance folks.

(Don't worry too much about the differences between SLES15-Migration-1.15.1-8.x86_64.rpm and SLES15-Migration-1.15.2.ses.0-15.x86_64.rpm, I don't think there's anything critical in the newer version that materially affects this test)